### DISCUSSION TITLE
Video size varies

### DISCUSSION THREAD

**Discussion by Jan542** [aclap-dev/video-downloadhelper#1006] [Help and Q&A]
### Your question

Dear Q&A Community, 

i purchased a license today to remove the watermarks and that worked just fine, but now the same video i just downloaded with the exact same version of VDH and the exact same settings regarding quality is much smaller.
Is some information missing, or why is that? 

I downloaded as .mp4
Is that even a good choice? 
Thank you very much in Advance!!

### Addon & CoApp version and details

version: 9.0.2.12
target: mozilla
channel: stable
lang: de
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"1127....."}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:128.0) Gecko/20100101 Firefox/128.0
{
    "valid_license_message_has_been_displayed": true,
    "http_media_download_strategy": "inbrowser",
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\My Name\\Downloads",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\My Name\\dwhelper"
        }
    },
    "show_thumbnail_in_notification": false,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "v9_panel_error_report_button2": "Bericht",
        "v9_panel_error_reported_button": "Bericht gesendet, Danke!",
        "v9_panel_error_unknown_description": "Leider ist das Addon auf einen unerwarteten Fehler gestoßen. Es wäre sehr hilfreich, wenn Sie diesen Fehler melden könnten (er ist anonym!)."
    },
    "yt_warning": false,
    "use_wide_ui": true,
    "use_sidebar": false,
    "open_count_store": 24,
    "successfull_dl": 7
}

**Comment by GrampaWildWilly** [comment]
Give me the URLs you're talking about.  I'll download the videos & post what I get.  Then we can compare results.  At least, if the content doesn't require a log in.  If it does, I probably can't help you.
